documentation/content/en/books/handbook/basics/_index.adoc

```diff
@@ -82,7 +82,7 @@ login:
 ....
 
 The first line contains some information about the system.
-The `amd64` indicates that the system in this example is running a 64-bit version of FreeBSD.
+The `amd64` indicates that the system in this example is running an x86 64-bit version of FreeBSD.
```

Comment:
```suggestion
`amd64` indicates x86, 64-bit.
```
The [existing context](https://docs.freebsd.org/en/books/handbook/basics/#consoles):

<details>
<summary>screen shot</summary>

![image](https://github.com/freebsd/freebsd-doc/assets/192271/59189c3a-6572-4523-919f-0f98ad34b07a)

</details>

* a FreeBSD site
* a FreeBSD document.
* the _FreeBSD/_ part of `FreeBSD/amd64` needs no explanation. 

https://people.freebsd.org/~grahamperrin/style/writing/tips/ | be concise :-)